cachi2/core/models/sbom.py

```diff
@@ -11,6 +11,15 @@
     "cdx:npm:package:development",
 ]
 
+HashAlgorithm = Literal["SHA-256", "SHA-384", "SHA-512"]
```

Comment:
I left this list compact, but more algorithms should be added as the need arises.
https://cyclonedx.org/docs/1.5/json/#components_items_hashes_items_alg